src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -26,6 +30,9 @@
 	@Option( names = "--help", usageHelp = true, hidden = true )
 	private boolean displayHelp;
 
+	@Option( names = "--exclude", description = "Ignore matching elements during accept." )
```

Comment:
Is `--exclude` fixe? Maybe stick to `--ignore` or so? CC @roesslerj

Reply:
Description suggestion: "Filter(s) to exclude changes from the diff." (And maybe adapt "exclude" if we choose another param name.)

Reply:
Oh and could adapt the description as mentioned above?